mail_connector_queue/models/mail_mail.py

```diff
@@ -2,11 +2,18 @@
 # Copyright 2018 ACSONE SA/NV
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from openerp import fields, models
+from openerp import fields, models, api
 
 
 class MailMail(models.Model):
 
     _inherit = 'mail.mail'
 
-    priority = fields.Integer()
+    mail_job_priority = fields.Integer(oldname='priority')
+
+    @api.model
+    def create(self, vals):
+        context_value = self.env.context.get('default_mail_job_priority')
+        if context_value and not vals.get('mail_job_priority'):
+            vals['mail_job_priority'] = context_value
+        return super(MailMail, self).create(vals)
```

Comment:
@andreparames Is this override actually necessary? I'd say Odoo does exactly that out of the box.

Reply:
Hm, you need to put in vals so the event handlers see it, right? But then the event handler could do a browse to see what priority has actually been saved. Would that not be simpler? Not sure, I'm not familiar with these event handlers.

Reply:
You're right, I see no reason not to fetch from the record by doing a browse. I'll rewrite.